### PR TITLE
New events: Federal Crowdsourcing Webinar Series, Episode 10: A preci…

### PR DESCRIPTION
**Pull request by DanielJPino-ux**
…sionFDA Strike for Public Health

Adding new event: Federal Crowdsourcing Webinar Series: Episode 10 to digital.gov page

A one-line description of the changes you're making and how they'll affect users.

---

**Preview:**

### CODE REVIEW

**Comment by DanielJPino-ux** [comment]
@saracope  - this had an unsuccessful build on federalist; can you take a look? Thanks!!

also speakers: Jarah Meador, Holly Stephens (#2504) and Elaine Johanson (#2503) are speakers on the event and need to be tagged in the event itself

File: content/events/2020/05/2020-05-12-federal-crowdsourcing-webinar-series-episode-10.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+host: "Challenge.gov "
+event_organizer: "Digital.gov"
+registration_url: https://www.eventbrite.com/e/federal-crowdsourcing-webinar-series-episode-10-a-precisionfda-strike-for-public-health-registration-103947748514
+captions: 
+
+# start date
+date: 2020-05-12 14:000:00 -0500
+
+# end date
+end_date: 2020-05-12 15:000:00 -0500
```
Comment (saracope) [inline content/events/2020/05/2020-05-12-federal-crowdsourcing-webinar-series-episode-10.md:17]: remove the extra 0

File: content/events/2020/05/2020-05-12-federal-crowdsourcing-webinar-series-episode-10.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+title: "Federal Crowdsourcing Webinar Series, Episode 10: A precisionFDA Strike for Public Health"
+deck: ""
+summary: "See how the U.S. Food and Drug Administration crowdsources to identify potential health risks."
+host: "Challenge.gov "
+event_organizer: "Digital.gov"
+registration_url: https://www.eventbrite.com/e/federal-crowdsourcing-webinar-series-episode-10-a-precisionfda-strike-for-public-health-registration-103947748514
+captions: 
+
+# start date
+date: 2020-05-12 14:000:00 -0500
```
Comment (saracope) [inline content/events/2020/05/2020-05-12-federal-crowdsourcing-webinar-series-episode-10.md:14]: remove the extra 0 in this time

**Comment by saracope** [comment]
Federalist is showing this error: "unable to parse date: 2020-05-12 15:000:00 -0500"
It looks like an extra zero got added in each of the timestamps. Updating those should fix it.

**Comment by DanielJPino-ux** [comment]
@saracope - this looks good to merge! Thanks!!

**Comment by saracope** [comment]
This one can be completed after #2503 and #2504 are merged.

**Pull request merged**